src/timer/timer_game_calendar.h

```diff
@@ -7,11 +7,11 @@
 
 /** @file timer_game_calendar.h Definition of the game-calendar-timer */
 
+#include "stdafx.h"
```

Comment:
Please never ever include anything before the header protection.

For all header files: copyright, `/** */`, header-protection, includes, rest :)